src/Core/src/ImageSources/FontImageSourceService/FontImageSourceService.Android.cs

```diff
@@ -7,13 +7,58 @@
 using Android.Graphics.Drawables;
 using Android.Util;
 using Bumptech.Glide;
+using Bumptech.Glide.Load;
+using Bumptech.Glide.Load.Engine;
+using Bumptech.Glide.Load.Resource.Bitmap;
+using Bumptech.Glide.Request;
+using Bumptech.Glide.Request.Target;
 using Microsoft.Extensions.Logging;
 using Microsoft.Maui.BumptechGlide;
 
 namespace Microsoft.Maui
 {
 	public partial class FontImageSourceService
 	{
+		public override async Task<IImageSourceServiceResult<Drawable>?> LoadDrawableAsync(IImageSource imageSource, Android.Widget.ImageView imageView, CancellationToken cancellationToken = default)
+		{
+			if (imageSource is IFontImageSource fontImageSource)
+			{
+				if (fontImageSource.IsEmpty)
+					return null;
+
+				var glyph = fontImageSource.Glyph;
+
+				var size = FontManager.GetFontSize(fontImageSource.Font);
+				var unit = fontImageSource.FontAutoScalingEnabled ? ComplexUnitType.Sp : ComplexUnitType.Dip;
+				var textSize = TypedValue.ApplyDimension(unit, size.Value, imageView.Context?.Resources?.DisplayMetrics);
+				var typeface = FontManager.GetTypeface(fontImageSource.Font);
+				var color = (fontImageSource.Color ?? Graphics.Colors.White).ToPlatform();
+
+				try
+				{
+					var listener = new RequestBuilderExtensions.RequestCompleteListener();
+					var glide = Glide.With(imageView.Context);
+					var builder = glide
+						.Load(new FontImageSourceModel(glyph, textSize, typeface, color))
+						.AddListener(listener)
+						.Override(Target.SizeOriginal, Target.SizeOriginal);
```

Comment:
This tells Glide to set the imageview to the size of the original bitmap we generate - otherwise it tries to fill the device screen width space and we end up with a blurry image.